phpseclib/Crypt/Hash.php

```diff
@@ -140,7 +180,11 @@ function setHash($hash)
             case 'sha1':
                 $this->length = 20;
                 break;
+            case 'sha512/224':
```

Comment:
'sha224' has also been added (see https://github.com/phpseclib/phpseclib/pull/956/files#diff-b4b2acaebb5e3d125481ec5da11baff5R1972), but its length is not set here. Is this on purpose?